dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -1275,6 +1364,68 @@ fn cmd_load(
     Ok(Some(s))
 }
 
+fn cmd_load_example(
+    sim: &mut ReconfiguratorSim,
+    args: LoadExampleArgs,
+) -> anyhow::Result<Option<String>> {
+    if sim.needs_reset_for_load_example() {
+        bail!("system is not empty; call 'reset' before loading an example");
```

Comment:
Should this say something like: "You have made changes to the simulated system.  Use `reset` to wipe these before loading an example system."

Reply:
How about "changes made to simulated system: run `wipe system` before loading an example system"

Huh, I guess I'm personally inclined against using "you" in CLI text. Never thought about this before.

Reply:
Made this change.